recipe/meta.yaml

```diff
@@ -27,16 +26,15 @@ requirements:
     - {{ cdt('libxext') }}           # [linux]
   host:
     - python
-    - numpy =1.16  # [py < 39]
-    - numpy =1.19  # [py >= 39]
+    - numpy
```

Comment:
For `numpy`, there is a global pinning for that package for each Python version in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml. I remember the pinning used to follow 1.16 for Python versions earlier than 3.9 and 1.19 for 3.9 and later, but it looks like it is 1.19 for all versions of Python now.
So yes, the global pinnings mean that this package will only be compatible with `numpy >= 1.19`. If there is some package that requires an older version of `numpy`, that could be an issue. However, new builds of other packages should also follow the same pinning, so hopefully there are not many packages that only work with earlier versions of `numpy`.
Are there legacy plugins to PyMOL that require an earlier version of `numpy`? If so, I can add back the earlier pinning.

Reply:
I do not know any plugins that require an earlier version, but I am not familiar with a lot of PyMOL plugins. Since NumPy uses semantic versioning, I do not think that there are incompatibilities.

I suspected that without the pinning only the most recent NumPy version is used, but I was unaware of the global pinning, so this change is fine for me.